hermeto/core/package_managers/cargo/main.py

```
@@ -133,7 +165,7 @@ def to_component(self) -> Component:
         return Component(name=self.name, version=self.version, purl=self.purl.to_string())
 
 
-def fetch_cargo_source(request: Request) -> RequestOutput:
+def fetch_cargo_source(request: Request, invoked_through_pip: bool = False) -> RequestOutput:
```

Ruff (0.16.0)

[warning] Boolean-typed positional argument in function definition

(FBT001)